client/state/action-types.js

```diff
@@ -175,6 +175,9 @@ export const JETPACK_MODULE_ACTIVATE_FAILURE = 'JETPACK_MODULE_ACTIVATE_FAILURE'
 export const JETPACK_MODULE_DEACTIVATE = 'JETPACK_MODULE_DEACTIVATE';
 export const JETPACK_MODULE_DEACTIVATE_SUCCESS = 'JETPACK_MODULE_DEACTIVATE_SUCCESS';
 export const JETPACK_MODULE_DEACTIVATE_FAILURE = 'JETPACK_MODULE_DEACTIVATE_FAILURE';
+export const JETPACK_MODULE_LIST_REQUEST = 'JETPACK_MODULE_LIST_REQUEST';
+export const JETPACK_MODULE_LIST_REQUEST_SUCCESS = 'JETPACK_MODULE_LIST_REQUEST_SUCCESS';
+export const JETPACK_MODULE_LIST_REQUEST_FAILURE = 'JETPACK_MODULE_LIST_REQUEST_FAILURE';
```

Comment:
This is your call, of course, but if you ask me, I'd probably name those:

```
export const JETPACK_MODULES_RECEIVE = 'JETPACK_MODULES_RECEIVE';
export const JETPACK_MODULES_REQUEST = 'JETPACK_MODULES_REQUEST';
export const JETPACK_MODULES_REQUEST_FAILURE = 'JETPACK_MODULES_REQUEST_FAILURE';
export const JETPACK_MODULES_REQUEST_SUCCESS = 'JETPACK_MODULES_REQUEST_SUCCESS';
```

Notice that there are 4 action types, not 3. The idea here is to differentiate the action that indicates request success/failure from the action that receives the data (in this case, the module data). If you have a look at the rest of the actions that end in `_RECEIVE`, you'll see that's being done this way consistently throughout Calypso. See my line comment in actions.js for a demo of how I envision these 4 action types to work.